React Props: Greeting Challenge (#137)
neuefische/web-exercises: sessions/react-props/greeting/README.md

# Greeting

> With React, you can build components and reuse them as often as you like. However, you sometimes want to slightly adjust the component according to its environment, e.g. to use a different text to be displayed.

Reviewer: ```suggestion
In React, you can build components and reuse them as often as you like. However, sometimes you want your components to be able to adapt to their environment, or other conditions, e.g. to have them display different types of content.
```

Let us use `props` to make components totally reusable!

## Task

### Create a Component with props

> Look at the [`src/App.js`](./src/App.js) file: the `App` component returns a heading which wants to be replaced by a component. Your task is to write a `Greeting` component which renders a greeting according to its prop.

Reviewer: ```suggestion
Look at the [`src/App.js`](./src/App.js) file: the `App` component returns a default heading element. Replace this element with a custom `Greeting` component, which renders a greeting according to its props.
```
